src/core/wee-config.c

```diff
@@ -3402,6 +3421,22 @@ config_weechat_init_options ()
         &config_check_separator, NULL, NULL,
         &config_change_buffers, NULL, NULL,
         NULL, NULL, NULL);
+    config_look_status_more_prefix = config_file_new_option (
+        weechat_config_file, ptr_section,
+        "status_more_prefix", "string",
+        N_("text to display before count for buffer with new data (status bar)"),
+        NULL, 0, 0, _("-MORE("), NULL, 0,
```

Comment:
I'm not sure if this `_("-MORE(")` as a default value is still proper if we want to allow it to be translatable but modifiable by the user if they wish to do so.